src/components/footer/style.module.css

```
@@ -1,9 +1,79 @@
 .root {
-  padding: 1rem;
-  min-height: 160px;
-  background: var(--token-color-palette-neutral-700);
+  align-items: center;
+  background: var(--token-color-palette-neutral-0);
   color: var(--token-color-palette-neutral-600);
   display: flex;
-  align-items: center;
-  justify-content: center;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  padding: 16px 0;
+  position: relative;
+
+  /* before element acts as full-width border,
+     no matter the container size*/
+  &::before {
+    background: var(--token-color-neutral-border-primary);
+    content: '\200B';
+    height: 1px;
+    left: 0;
+    left: 50%;
+    overflow: hidden;
+    position: absolute;
+    top: 0;
+    transform: translateX(-50%);
+    width: 100%;
+    width: 200vh;
```

The first declaration shouldn't be necessary I think.
```suggestion
    width: 200vh;
```